.env.example

```
@@ -113,3 +113,7 @@ RAZORPAY_WEBHOOK_SECRET=
 # FLEET_MANAGER_LAMBDA_NAME=veolms-fleet-manager
 # FLEET_MANAGER_LAMBDA_REGION=us-east-1
 FLEET_MANAGER_HEARTBEAT_SECONDS=10
+
+
+# TRUST_PROXY=false
+TRUST_PROXY=true # if using TRUST_PROXY
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)